pkgs/development/python-modules/imbalanced-learn/default.nix

```diff
@@ -60,19 +63,33 @@ buildPythonPackage rec {
 
   preCheck = ''
     export HOME=$TMPDIR
+    # The GitHub source contains too many files picked up by pytest like
+    # examples and documentation files which can't pass.
+    cd $out/${python.sitePackages}
   '';
 
   disabledTestPaths = [
     # require tensorflow and keras, but we don't want to
     # add them to nativeCheckInputs just for this tests
     "imblearn/keras/_generator.py"
+    "imblearn/keras/tests/test_generator.py"
```

Comment:
maybe these should be directory matches if we'd like to skip all keras tests (same for tensorflow)

Reply:
I have a faint recollection of also thinking that when i read the comment, trying that, and it not working, but I can quickly take another look at it.

Reply:
seems to work fine with dir match not sure what i was remembering